Resources/Prototypes/_LostParadise/Maps/NW/NY_Plot_113.yml

```
@@ -0,0 +1,74 @@
+      stationProto: StandardNanotrasenStation
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: '{0} Участок 113 {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: 'DV'
+        - type: GridSpawn
+          groups:
+            mining:
+              paths:
+              - /Maps/_LostParadise/Shuttles/Salvage_113.yml
```

yamllint (1.35.1)

[error] wrong indentation: expected 16 but found 14

(indentation)